inst/assets/book.scss

```diff
@@ -168,6 +168,200 @@ li.dropdown-header > span.menu-text {
   -webkit-justify-content:flex-start;
 }
 
+/* --------------- Introductory workflow ---------------- */
+
+#workflow {
+
+	// Other styles
+
+	#banner {
+		display: flex;
+		justify-content: center; 
+		align-items: center; 
+		max-width: 70%;
+		max-height: auto;
+		text-align: center; 
+		margin: 0 auto; 
+	}
+
+	#quarto-content{
+	  margin: 0;
+	  padding: 0;
+	  width: 100%;
```

Comment:
What these does?

Reply:
Why the default is not working?

Reply:
> What these does?

It's to give a different style to the workflow

Reply:
OK good, does it affect other sections?